src/app/services/theme/theme.service.ts

```diff
@@ -62,6 +62,8 @@ export class ThemeService {
     for (const themeColor of Object.keys(this.theme)) {
       this.setColor(themeColor as ThemeColor, this.theme[themeColor]);
     }
+
+    this.overrideBootstrapColors(this.theme.highlight);
```

Comment:
do we need to special case the SSR stuff? 

Or are the visual differences too small to worry about. Leave a note if we're intentionall ignoring.